tooling/cli/node/tauri.js

```diff
@@ -20,7 +20,7 @@ if (bin === '@tauri-apps/cli') {
 }
 // Even if started by a package manager, the binary will be NodeJS.
 // Some distribution still use "nodejs" as the binary name.
-else if (binStem.match(/(nodejs|node)\-?([0-9]*)*$/g)) {
+else if (binStem.match(/(nodejs|node|bun)\-?([0-9]*)*$/g)) {
```

Comment:
Running a script with `bunx --bun` will execute it with the `bun` executable even if the shebang says `node`